docs/manuals/source/TasksAndConcepts/CatalogMaintenance.rst

```diff
@@ -23,16 +23,13 @@ dbconfig-common (Debian)
    single: Platform; Debian; dbconfig-common
    single: Platform; Ubuntu; dbconfig-common
 
-Since Bareos :sinceVersion:`14.2.0: dbconfig-common (Debian)` the Debian (and Ubuntu) based packages support the **dbconfig-common** mechanism to create and update the Bareos database, according to the user choices.
+Since Bareos :sinceVersion:`14.2.0: dbconfig-common (Debian)` the Debian / Ubuntu / UCS packages support the **dbconfig-common** mechanism to create and update the Bareos database, according to the user choices.
 
 The first choice is, if **dbconfig-common** should be used at all. If you decide against it, the database must be configured manually, see :ref:`CatMaintenanceManualConfiguration`.
 
 If you decided to use **dbconfig-common**, the next question will be asked.
 
 .. image:: /include/images/dbconfig-1-enable.*
-   :width: 45.0%
-
-
 
 The **dbconfig-common** configuration (and credentials) is done by the **bareos-database-common** package. Settings are stored in the file :file:`/etc/dbconfig-common/bareos-database-common.conf`. If you need to repeat this step, you can use the :command:`dpkg-reconfigure bareos-database-common` command.
 
```

Comment:
You want this change in the Debian/dbconfig section? For Debian, the `grant_bareos_privileges` is not required.

Reply:
Ok so the warning block (line 40+) is only about Debian and Bareos < 14.2 ?

What I don't like too much, is already receive comment about people not running the grant because they have seen this one.

Maybe we should remove the remark about the version, and move the section on manual update, because there, it is really advised to do the grant each time.

What's your thought?

Reply:
I removed the paragraph. However, if your database configuration started in bareos < 14.2, then calling this script will required after every update that adds a table.